api/graphql/schema.graphql

```diff
@@ -628,7 +627,7 @@ type Query {
     me: User @auth
     user(id: ID! @eq): User @find @guard @can(ability: "view", query: true)
     users: [User]! @all @guard @can(ability: "viewAny")
-    usersPaginated(where: UserFilterAndOrderInput): [User]! @paginate(defaultCount: 10, maxCount: 1000 scopes: ["order"]) @guard @can(ability: "viewAny")
+    usersPaginated(where: UserFilterInput orderBy: [OrderByClause!] @orderBy): [User]! @orderBy(column: "id", direction: ASC) @paginate(defaultCount: 10, maxCount: 1000 ) @guard @can(ability: "viewAny")
```

Comment:
I mostly added the default ordering on id to ensure that the contents of each page was stable when you refresh the page and it is indexed column for good db performance.  I could switch to created_at but it will be much less stable when the user hasn't explicitly asked for any sorting yet.  If you're sure, I should probably add a db migration with an index for that column.

Reply:
Added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3204/commits/4121bf10afab52a36d47d1bf40456d510ce2a21a

Reply:
Indexing the column is an excellent point! I do think this is a better default. When we consider the final table, it might make sense to index other columns for sorting as well.